src/uwtools/resources/jsonschema/fv3.jsonschema

```diff
@@ -261,9 +200,6 @@
         "run_dir"
       ],
       "type": "object"
-    },
-    "user": {
-      "type": "object"
```

Comment:
I don't think we need this. Our drivers will validate each top-level config block that they use (e.g. the FV3 driver validates the `fv3` and `platform` blocks), and anything else in the YAML config is fair game for users. See e.g. the example config file attached to to PR and how it DRYs out `sfc_climo_gen` input filenames with Jinja2 references.